adoc/deployment-bare-metal.adoc

```diff
@@ -0,0 +1,73 @@
+include::entities.adoc[]
+:isofile: SLE-15-SP1-Installer-DVD-x86_64-Build225.1-Media1.iso
+:isolink: http://download.suse.de/install/SLE-15-SP1-Installer-GMC1/
```

Comment:
we shouldn't refer to internal server, anyway GMC1 is outdated now, current is http://download.suse.de/install/SLE-15-SP1-Installer-GMC3/SLE-15-SP1-Installer-DVD-x86_64-GMC3-DVD1.iso

Reply:
How should we do it? Remove the iso from autoyast and update docs? Or how should we do it?

Reply:
The official DVD can be download from https://www.suse.com/betaprogram/sle-beta/

Reply:
Those are outdated values for the attributes. If you give me new URLs I can use those. The places that actually refer to the downloads already point to the sle beta page.

Reply:
Now we refer to official SLES deployment guide so this is not needed anymore.